### DISCUSSION TITLE
Craft wearables [problem of hats]

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#4016] [Ideas]
The idea is to give the player the ability to transfer boost from one item to another. 
Crafting would cost SFL or, better yet, Block Bucks.
Example: 
Transfer a reinforcement from Tofu Mask to Moon Hat, Tofu Mask will burn and Moon Hat will get an extra reinforcement. 

**Advantages:**
1. Solves the problem of wearable items, no need to add an infinite number of cones. 
2. Burning SFL or extra income for the team (when using Block Bucks). 
3. carrying your favorite item 

**Disadvantages:**
1. Withdrawal problem
2. Boost display on OpenSea. 

_I'm sure many will say we should just stop adding hats to the game, but that's not a solution to the problem, as we'll still end up in a stalemate with adding new wearable items._

**Comment by ShinKan42** [comment]
This will generate tons of NFTs, hard to manage.
I would rather like launching a crafting system to craft different tiers of items with lower and minor boosts which is largely porpotioned.

**Reply by Dionis404** [reply]
I completely agree that this system will generate a lot of NFT tokens. 
I just don't have any other ideas
But the system is necessary because we will in any case achieve that the wearable will be more than bumpkins on the farm.

**Reply by ShinKan42** [reply]
> I completely agree that this system will generate a lot of NFT tokens. I just don't have any other ideas But the system is necessary because we will in any case achieve that the wearable will be more than bumpkins on the farm.

I'm against it because it will either be **Infinite Combinations with Infinite Bugs** or **Finite Combinations which is against the whole idea.
What you are addressing is genuine, but this particular method is not getting there at all.

### Crafting
So still, just craft a higher new tier item with a brand new effect with lower ones.
In this way, the lower items will get active traffic, eventually making items scarce and up in price.
Then we can see Massive Minor Boosts and Elite Strong Boosts.

For all-round players, they will need to obtain again the Minor ones.
This can address the problem that some cheap items are not attractive at all with 1-10Ks mints.
